src/validate/mod.rs

```diff
@@ -22,8 +22,6 @@ lazy_static! {
     static ref SENDER_NAME_FORMAT: Regex =
         Regex::new("^[A-Za-z0-9-]+(?: [A-Za-z0-9-]+)*$").unwrap();
     static ref SENDGRID_API_KEY_FORMAT: Regex = Regex::new("^[A-Za-z0-9._-]+$").unwrap();
-    static ref SQS_URL_FORMAT: Regex =
-        Regex::new("^https://sqs\\.[a-z0-9-]+\\.amazonaws\\.com/[0-9]+/[A-Za-z0-9-]+$").unwrap();
```

Comment:
Oh, you were moving it over, not duplicating it, sorry. I still think it's best kept over here though because it makes it much easier to write test cases.

Reply:
(and I quite like how all our validation code lives in the same place, it is especially useful for things like email addresses that are validated both in settings and in request params)

Reply:
I agree with leaving this stuff inside validation, seems cleaner.